assignment 5C/PlaceQueen.java

```diff
@@ -0,0 +1,235 @@
+package Assignment5C;
+
+/**
+ * This class implement method to solve N Queen problem
+ * @author Sabir
+ *
+ */
+public class PlaceQueen {
+	
+	public static int queenPosition[][];
+	
+	/**
+	 * 
+	 * @param chessBoard two dimension NxN integer array containing zeros
+	 * @param startRow positive integer value indicating starting row
+	 * @param dimensionOfMatrix dimension of two dimension NxN matrix
+	 * @return true if their is solution to place all queen safely else false
+	 */
+	public boolean placeNQueen(int chessBoard[][],int startRow, int dimensionOfMatrix){
+		queenPosition = chessBoard;
+		boolean success = nQueen(chessBoard,startRow,dimensionOfMatrix);
+		chessBoard = queenPosition;

```

Comment:
Do not use global variable like this until common value required. Here you change the value in other method and use in other. 
You can use reference or pointer variable to update parameter value in called method.